frontend/src/modules/CrudModule/index.jsx

```diff
@@ -110,8 +111,13 @@ function FixHeaderPanel({ config }) {
             size="small"
           ></Button>
         </Col>
-        <Col className="gutter-row" span={22}>
+        <Col
+          className="gutter-row"
+          style={{ display: 'flex', justifyContent: 'space-between' }}
+          span={22}
+        >
           <h1 style={{ fontSize: 20, marginBottom: 20 }}>{config.PANEL_TITLE}</h1>
+          <CloseOutlined onClick={collapsePanel} style={{ marginBottom: 20 }} />
```

Comment:
@rkmdCodes please replace close icon with <MenuOutlined />

Reply:
hi @salahlalami sir
thank you for review,
can you please specify which icon i use to replace close icon

Reply:
@rkmdCodes use this one <MenuFoldOutlined />

Reply:
@salahlalami the icon does not show do you mean <MenuOutlined />

Reply:
@polymahh  @rkmdCodes I mean this one  MenuFoldOutlined  ,

Reply:
hi @salahlalami @polymahh i just updated the icon